hw_02.js

```diff
@@ -0,0 +1,100 @@
+// 1. Задача про обчислення різниці часу.
+// Напишіть функцію яка буде буде приймати 3 параметри
+// — початкову дату (string)
+// — кінцеву дату (string)
+// — розмірність ('days', 'hours', 'minutes', seconds)
+// Функція повертатиме часовий період між цими датами згідно розмірності.
+// Також вкажіть значення по замовчуванню для всіх цих параметрів (на ваш вибір).
+// Функція має коректно працювати навіть якщо початкова дата пізніше ніж кінцева дата.
+
+function getTimePeriod(startDate = new Date(), endDate = new Date(), unit = 'days') {
+    startDate = new Date(startDate);
+    endDate = new Date(endDate);
+
+    const earlierDate = startDate < endDate ? startDate : endDate;
+    const laterDate = startDate < endDate ? endDate : startDate;
+
+    const timeDifference = laterDate - earlierDate;
+
+    switch (unit) {
+      case 'days':
+        return timeDifference / (24 * 60 * 60 * 1000);
+      case 'hours':
+        return timeDifference / (60 * 60 * 1000);
+      case 'minutes':
+        return timeDifference / (60 * 1000);
+      case 'seconds':
+        return timeDifference / 1000;
```

Comment:
```suggestion
        return timeDifference / 1000;
```
Якщо у всіх умовах / 1000, то є сенс винести в одне місце, наприклад туди де обчислюється timeDifference

Reply:
Винесла, зробила ` const timeDifference = (laterDate - earlierDate) / 1000;`